server/plugins/task-plugin-ob-mysql/src/main/java/com/oceanbase/odc/plugin/task/obmysql/datatransfer/task/BaseObLoaderDumperTransferTask.java

```diff
@@ -0,0 +1,261 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.oceanbase.odc.plugin.task.obmysql.datatransfer.task;
+
+import static com.oceanbase.odc.core.shared.constant.OdcConstants.DEFAULT_ZERO_DATE_TIME_BEHAVIOR;
+import static com.oceanbase.tools.loaddump.common.constants.Constants.JdbcConsts.JDBC_URL_USE_SERVER_PREP_STMTS;
+import static com.oceanbase.tools.loaddump.common.constants.Constants.JdbcConsts.JDBC_URL_ZERO_DATETIME_BEHAVIOR;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+import org.apache.commons.collections4.CollectionUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.oceanbase.odc.core.shared.Verify;
+import com.oceanbase.odc.plugin.task.api.datatransfer.DataTransferCallable;
+import com.oceanbase.odc.plugin.task.api.datatransfer.model.DataTransferTaskResult;
+import com.oceanbase.odc.plugin.task.api.datatransfer.model.ObjectResult;
+import com.oceanbase.tools.loaddump.common.enums.DataFormat;
+import com.oceanbase.tools.loaddump.common.enums.ObjectType;
+import com.oceanbase.tools.loaddump.common.model.BaseParameter;
+import com.oceanbase.tools.loaddump.common.model.DumpParameter;
+import com.oceanbase.tools.loaddump.common.model.ObjectStatus;
+import com.oceanbase.tools.loaddump.common.model.TaskDetail;
+import com.oceanbase.tools.loaddump.context.TaskContext;
+import com.oceanbase.tools.loaddump.manager.session.SessionProperties;
+
+import lombok.NonNull;
+
+/**
+ * {@link BaseObLoaderDumperTransferTask}
+ *
+ * @author yh263208
+ * @date 2022-07-25 14:41
+ * @since ODC_release_3.4.0
+ */
+public abstract class BaseObLoaderDumperTransferTask<T extends BaseParameter> implements DataTransferCallable {
```

Comment:
there is no need to point out ob-loader-dumper, `BaseOBTransferTask` will be better?

Reply:
renamed to BaseOceanBaseTransferJob